backend/typescript/services/implementations/reviewService.ts

```diff
@@ -386,6 +386,31 @@ class ReviewService implements IReviewService {
 
     return result;
   }
+
+  async updateReviews(id: number, entity: ReviewRequestDTO): Promise<void> {
+    try {
+      const updatedReview = await PgReview.update(
+        {
+          body: entity.body,
+          byline: entity.byline,
+          featured: entity.featured,
+          books: entity.books,
```

Comment:
looks like you'll have to update the tables individually. From my testing, I could stuff on the review table just fine like body and featured but then if i tried to update a tag or anything in a book, it didn't update. I'm assuming you also have to call PgBook.update and PgTag.update etc